crypto/asn1/f_int.c

```diff
@@ -84,9 +84,9 @@ int a2i_ASN1_INTEGER(BIO *bp, ASN1_INTEGER *bs, char *buf, int size)
             /*
              * This #ifdef is not strictly necessary, since the characters
              * A...F a...f 0...9 are contiguous (yes, even in EBCDIC - but
-             * not the whole alphabet). Nevertheless, isxdigit() is faster.
+             * not the whole alphabet). Nevertheless, ossl_isxdigit() is faster.
```

Comment:
How about acting upon the "not strictly necessary" and removing it? Less #ifdef is better.